vagrant-post-script/orchestrate_docker.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 
 cd oms-docker/docker
-docker-compose up -d
-echo "############# ok until here"
-docker-compose logs -f omscore-bootstrap
-docker-compose logs -f omsevents-bootstrap
-echo "############# done"
+make bootstrap
```

Comment:
I don't think it is hard to maintain and it has anyway a broader scope: you can tell any illiterate to install virtualbox and vagrant and to download the zip, and they can check the intranet on their laptop -> useful for demos

But that's beyond the scope of this comment, to get back in topic I could say: look [here](https://github.com/linuxbandit/oms-docker/blob/feat-swarm_ready/docker/docker-compose.yml#L311) and notice that a full path is needed to mount the code (~ doesn't work). Clearly vagrant standardises it. It is just faster to get one started without hassle, because I agree with nico that images are immutable anyway and give us enough stability (except the docker daemon being critical)